turbo.json

```diff
@@ -13,16 +13,17 @@
       "outputs": ["dist/**"],
       "cache": false
     },
-    "build:core": {
+    "build:wallet": {
       "outputs": ["dist/**"],
       "cache": false
     },
     "build:ui": {
       "outputs": ["dist/**"],
+      "dependsOn": ["^build:common"],
       "cache": false
     },
-    "build:wallet": {
-      "outputs": ["dist/**"],
+    "build:core": {
+      "outputs": ["dist/**", "^build:common", "^build:wallet"],
```

Comment:
Do we sure that outputs should include build scripts?

Reply:
this is misplaced 🤔